stories/types.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module '*.svg' {
+  const content: any;
```

Comment:
Why don't you type this as a string? Normally it would return either the url or the raw content